apps/organisations/models.py

```diff
@@ -1,32 +1,95 @@
 from autoslug import AutoSlugField
+from ckeditor.fields import RichTextField
+from ckeditor_uploader.fields import RichTextUploadingField
 from django.conf import settings
 from django.db import models
+from django.utils.translation import ugettext_lazy as _
 
-from apps.partners.models import Partner
+from adhocracy4 import transforms
+from adhocracy4.images import fields as images_fields
 
 
 class Organisation(models.Model):
     slug = AutoSlugField(populate_from='name', unique=True)
     name = models.CharField(max_length=512)
     initiators = models.ManyToManyField(
         settings.AUTH_USER_MODEL,
+        blank=True
+    )
+    title = models.CharField(
+        verbose_name=_('Title of your organisation'),
+        max_length=100,
+        default='Organisation',
+        help_text=_('The title of your organisation will be shown '
+                    'on the landing page. max. 100 characters')
+    )
+    description = models.CharField(
```

Comment:
I think we have to make most fields in here not required first or add a default value, right? Otherwise the migrations won't work on existing organisations

Reply:
They do have a default (`default='empty description'`), together with `preserve_default=False`, so the default is only applied during the migration.